mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/GlobalConfigRule.groovy

```diff
@@ -8,29 +8,20 @@ import com.avioconsulting.mule.linter.model.rule.RuleViolation
 
 class GlobalConfigRule extends Rule {
 
-    static final String RULE_ID = 'GLOBAL_CONFIG'
+    static final String RULE_ID = 'GLOBAL_CONFIG_EXISTS'
```

Comment:
If we follow the naming convention like Jenkins rule, we should also rename this rule to `GlobalConfigExistsRule`.